autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/ui/credential/management/AutofillCharacterValidator.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.ui.credential.management
+
+import com.duckduckgo.di.scopes.AppScope
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+interface AutofillCharacterValidator {
+    fun isLetter(character: Char): Boolean
+}
+
+@ContributesBinding(AppScope::class)
```

Comment:
This should be FragmentScoped.

Reply:
I have a shortcut for `@ContributesBinding` which defaults to `AppScope` and never think beyond the muscle memory required to invoke it whether it should be fragment or activity 😫